llvm/test/tools/llvm-exegesis/RISCV/latency-pre-assigned-register.s

```diff
@@ -0,0 +1,3 @@
+RUN: llvm-exegesis -mode=latency --benchmark-phase=assemble-measured-code -opcode-name=LB -mtriple=riscv64-unknown-linux-gnu
+
+CHECK: Warning: Pre-assigned register prevented usage of self-aliasing strategy.
```

Comment:
I am a bit confused by this test. What is the pre-assigned register that prevented usage of self-aliasing? I would expect a warning that no instructions were passed.

Reply:
This test was used previously, during implementing, now it is not used anymore, removed it, thanks for the catch.